packages/react-slider/src/components/Slider/Slider.types.ts

```diff
@@ -76,6 +76,12 @@ type SliderCommons = {
    */
   disabled?: boolean;
 
+  /**
+   * If true, the input will have inline display, allowing it be used within text content.
```

Comment:
I reaaaally have a hard time seeing a valid use case for a slider inline with text...but if it's in the design I guess we can have this? 🤷‍♀️

Reply:
![image](https://user-images.githubusercontent.com/1434956/155377708-b9bc86f1-6401-4234-8c3c-ae57bd010869.png)

it seems reasonable that someone would have this at some point. But yes, there is discussion on if this prop is really necessary. Right now it's for alignment across all of the inputs, and if we decide to remove it, we'll remove it everywhere.

Reply:
Ah okay, "inline" as in "label next to the slider" makes more sense. I was thinking of it like the scenario for Input where someone might put it inside a paragraph of text (which itself is bad for a11y but that doesn't stop people).

Reply:
Also, if there's discussion about whether a prop is necessary or not (and it's easy to achieve with one custom CSS style) we should probably lean towards NOT including it until if/when we decide it's necessary, since removing a prop is a breaking change.

Reply:
@spmonahan there's value in getting slider aligned with the other controls (adding inline), but I also agree that we might not want to add the prop if we plan to take it away later. Do you have a preference?

Reply:
Is there any urgency on getting the other aspects of this PR merged? I'm still collecting all the information on the different things we do today and I don't have a fully formed opinion on what we should align on yet. I should have a proposal for what to do here later this week so waiting seems like the simplest thing to do assuming that's okay by you.

Reply:
removed :D